app/uk/gov/hmrc/exports/mongock/changesets/ScalaCacheChangeLog.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.mongock.changesets
+
+import java.util.Map
+
+import com.github.cloudyrock.mongock.{ChangeLog, ChangeSet}
+import com.google.common.collect.ImmutableMap
+import com.mongodb.BasicDBObject
+import com.mongodb.client.MongoDatabase
+import org.bson.Document
+import uk.gov.hmrc.exports.services.CountriesService
+
+@ChangeLog
+class ScalaCacheChangeLog {
```

Comment:
Might be worth removing "Scala" from the name (?)

Reply:
Good point

Reply:
Changed